GNUmakefile

```diff
@@ -24,13 +24,15 @@ fieldalignment:
 	@echo "==> Analyzer structs and rearranged to use less memory with fieldalignment..."
 	fieldalignment -fix -test=false ./...
 
-download:
-	@echo Download go.mod dependencies
-	@go mod download
-
-install-tools: download
-	@echo Installing tools from tools.go
-	@go install $$(go list -f '{{range .Imports}}{{.}} {{end}}' tools.go)
```

Comment:
This command not working? Since you've changed it to hard coded installs

Reply:
Not works in Go 1.21 anymore. See jarvis-proto, there is it changed already too